src/openforms/formio/tests/test_api_fileupload.py

```diff
@@ -378,6 +378,40 @@ def test_cannot_connect_to_clamdav(self, m_config):
 
         self.assertEqual(0, len(tmpdir_contents))
 
+    def test_filename_with_leading_spaces(self):
+        self._add_submission_to_session(self.submission)
+
+        url = reverse("api:formio:temporary-file-upload")
+        file = SimpleUploadedFile(
+            "   my-file.txt", b"my content", content_type="text/plain"
```

Comment:
Probably it would be nice to add 2 subtests for  `   my-file.txt"` and  `my-file    .txt"` ?May be redundant but it covers all the possible variations of the filename.

Like we do here for example:
https://github.com/open-formulieren/open-forms/blob/master/src/openforms/formio/tests/test_validators.py#L30

Reply:
Check, yeah good idea. I've moved this to a subtest and added a `my-file    .txt` subtest